docs/report-entries.adoc

```diff
@@ -110,4 +110,32 @@ The extension will publish your report entry if the test fails.
 ON_ABORTED::
 The extension will publish your report entry if the test was aborted (for example because of an https://junit.org/junit5/docs/current/user-guide/#writing-tests-assumptions[Assumption]).
 
-Entries will always be published after the test finished.
\ No newline at end of file
+Entries will always be published after the test finished.
+
+=== Publishing test parameters
+
+You can access `@ParameterizedTest` parameter values and publish them with the `@ReportEntry` annotation.
+Add a number (starting from zero) between curly braces to reference the parameter with that index.
```

Comment:
Can you mention and link to the format you're using here? (Is it `String::format` or `MessageFormat`?)

Reply:
As far as I see - this is still open right?

Reply:
Did I misunderstand? I did mention that the format of the variables is in line with JUnit's `@ParameterizedTest`

Reply:
The way I understood Nicolais question is that he wanted to see you mention `String::format` or `MessageFormat`, but maybe I missunderstood him.